examples/digest/yarn.lock

```diff
@@ -0,0 +1,870 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Can this file be removed?

Reply:
done in https://github.com/cloudflare/workers-rs/pull/529/commits/8a50b68017e9e00d61aa3addd1134c62b23e0ec4